README.md

```diff
@@ -11,111 +16,148 @@
 
-1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
+- #### [Settings](https://baritone.leijurv.com/baritone/api/Settings.html#field.detail)
 
-# Getting Started
+- #### [Usage (chat control)](USAGE.md)
 
-Here are some links to help to get started:
+## Baritone API
 
-- [Features](FEATURES.md)
+There is extensive documentation for the Baritone API,
+[JavaDocs](https://baritone.leijurv.com/).
+Please note that usage of anything located outside the ``baritone.api`` package is unsupported by the API release jar.
 
-- [Installation & setup](SETUP.md)
+Below is an example of changing settings, and then pathing to an X/Z goal.
 
-- [API Javadocs](https://baritone.leijurv.com/)
+```
+BaritoneAPI.getSettings().allowSprint.value = true;
+BaritoneAPI.getSettings().primaryTimeoutMS.value = 2000L;
 
-- [Settings](https://baritone.leijurv.com/baritone/api/Settings.html#field.detail)
+BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
+```
 
-- [Usage (chat control)](USAGE.md)
+## FAQ
 
-## Stars over time
+*What inspired Baritone?*
 
-[![Stargazers over time](https://starchart.cc/cabaletta/baritone.svg)](https://starchart.cc/cabaletta/baritone)
+Baritone is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
+the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2
+through 1.16.5. Baritone focuses on reliability and particularly performance 
+(it's over[30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than
+MineBot at calculating paths).
 
-# API
+***
 
-The API is heavily documented, you can find the Javadocs for the latest release [here](https://baritone.leijurv.com/).
-Please note that usage of anything located outside of the ``baritone.api`` package is not supported by the API release
-jar.
+*How can I support Baritone?*
 
-Below is an example of basic usage for changing some settings, and then pathing to an X/Z goal.
+You can help support Baritone by contributing or by donating to **1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2**
 
-```
-BaritoneAPI.getSettings().allowSprint.value = true;
-BaritoneAPI.getSettings().primaryTimeoutMS.value = 2000L;
+***
 
-BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
-```
+*Can I use Baritone in my utility client?*
+
+Absolutely, as long as your usage is compliant
+with Baritone's [LGPLv3 license](https://github.com/cabaletta/baritone/blob/master/LICENSE).
 
-# FAQ
+More information about the LGPLv3 license can be found [here](https://www.gnu.org/licenses/lgpl-3.0.en.html).
 
-## Can I use Baritone as a library in my custom utility client?
+***
 
-That's what it's for, sure! (As long as usage complies with the LGPL 3.0 License)
+*When will support for Minecraft 1.17.~ be released?*
 
-## How is it so fast?
+Join the [Baritone Discord Server](https://discord.com/invite/s6fRBAUpmr) to be notified of future releases.
```

Comment:
pushed the discord server link a bit far down, it's practically hidden here. I think there should be a link closer to the top.